collector/src/benchmark/mod.rs

```diff
@@ -253,15 +253,18 @@ impl Benchmark {
             for (profile, prep_dir) in &profile_dirs {
                 let server = server.clone();
                 s.spawn::<_, anyhow::Result<()>>(move |_| {
+                    // Panic the thread if an error occurs to make sure that the whole scope will
```

Comment:
Would you like to rewrite this to a std::thread::scope while we're touching the code? I think we can drop the crossbeam-utils dependency as well.

Otherwise r=me.

Reply:
I will try it in another PR.